Library/Homebrew/requirements/java_requirement.rb

```diff
@@ -45,16 +30,69 @@ def inspect
 
   def display_s
     if @version
-      if @version[-1] == "+"
-        op = ">="
-        version = @version[0, @version.length-1]
-      else
+      if exact_version?
         op = "="
-        version = @version
+      else
+        op = ">="
       end
-      "#{name} #{op} #{version}"
+      "#{name} #{op} #{version_num}"
     else
       name
     end
   end
+
+  private
+
+  def version_num
```

Comment:
It gets rid of the "+" if it exists, i.e. if `@version` is "1.8+" it returns "1.8" and if `@version` is "1.8" it just returns that directly.